lib/mistral_ai_llm_controller_example/mistral_ai_llm_controller_page.dart

```diff
@@ -0,0 +1,212 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/logger_dialog.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/model.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/prompt.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/utils.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/mistral_client.dart';
+import 'package:mistralai_client_dart/mistralai_client_dart.dart';
+
+class MistralAiLlmControllerPage extends StatefulWidget {
+  const MistralAiLlmControllerPage({super.key});
+
+  @override
+  State<MistralAiLlmControllerPage> createState() =>
+      _MistralAiLlmControllerPageState();
+}
+
+class _MistralAiLlmControllerPageState
+    extends State<MistralAiLlmControllerPage> {
+  final TextEditingController commandInputController = TextEditingController();
```

Comment:
Don't forget to dispose it

Reply:
Resolved in [131237c](https://github.com/nomtek/flutter_ai_examples/pull/6/commits/131237c8c4ff02faddbe1f98214c6e83f62b6995)